plugins/katello/3.7/release_notes/release_notes.md

```diff
@@ -4,71 +4,36 @@ title: Release Notes
 version: 3.7
 ---
 
-# Katello 3.6 (Imperial IPA) Release Notes
+# Katello 3.7 (Blonde Ale) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.6/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.7/CHANGELOG.md).
 
-## Warning
+## Known RC1 Issues
+* Upgrades from 3.6 to 3.7 are not fully functional due to a failing database migration
```

Comment:
Should we reference redmine issues for the known issues that are being worked?

Reply:
Yes! Fortunately I think we can resolve the client tool repos prior to announcement which leaves one issue outstanding